app/src/main/java/org/jellyfin/androidtv/util/profile/LibVlcProfile.kt

```diff
@@ -73,17 +73,19 @@ class LibVlcProfile(
 			},
 			// Audio direct play
 			audioDirectPlayProfile(
-				CodecTypes.FLAC,
-				CodecTypes.AAC,
-				CodecTypes.MP3,
-				CodecTypes.MPA,
-				CodecTypes.WAV,
-				CodecTypes.WMA,
-				CodecTypes.MP2,
-				ContainerTypes.OGG,
-				ContainerTypes.OGA,
-				ContainerTypes.WEBMA,
-				CodecTypes.APE
+				arrayOf(
+					CodecTypes.FLAC,
+					CodecTypes.AAC,
+					CodecTypes.MP3,
+					CodecTypes.MPA,
+					CodecTypes.WAV,
+					CodecTypes.WMA,
+					CodecTypes.MP2,
+					ContainerTypes.OGG,
+					ContainerTypes.OGA,
+					ContainerTypes.WEBMA,
+					CodecTypes.APE
+				)
 			),
```

Comment:
```kotlin
			audioDirectPlayProfile(arrayOf(
				CodecTypes.FLAC,
				CodecTypes.AAC,
				CodecTypes.MP3,
				CodecTypes.MPA,
				CodecTypes.WAV,
				CodecTypes.WMA,
				CodecTypes.MP2,
				ContainerTypes.OGG,
				ContainerTypes.OGA,
				ContainerTypes.WEBMA,
				CodecTypes.APE
			)),
```

GitHub doesn't allow me do this one as a suggestion

Reply:
No worries, I tend to do these by hand to check they still build (as I've sometimes forgotten a bracket or something on a suggestion for someone else) :+1: